decentraland/unity-explorer
# style: create camera reel and camera mode buttons animations

## What does this PR change?
PR created to add the animations of camera reel and camera mode buttons on the side bar and main menu.

## How to test the changes?



1. Launch the explorer.
2. Check the sidebar and hover un-hover both buttons. Also press them. After going back to the world, validate the idle idle state of the buttons is normal (white outline).
3. Press tab to open the menu. Repeat: hover, un-hover and press the camera reel button (the camera mode doesn't appear here as is an in-world feature). Close the menu and open it back to check the idle state of the buttons is normal (grey outline).
